truss/vault_test.go

```diff
@@ -55,7 +55,7 @@ func TestVault(t *testing.T) {
 			Convey("returns keys as strings", func() {
 				list, err := vault.ListPath("kv/metadata")
 				So(err, ShouldBeNil)
-				So(list, ShouldResemble, []string{"foo"})
+				So(list, ShouldContain, "foo")
```

Comment:
Since we're running several tests against a single Vault instance concurrently, this now returns `[]string{"dir/", "file/", "foo"}`

Reply:
Nice and straightforward escape path, like it! Also the binary got some weight-loss by this :)